frontend/src/ui-kit/components/Search/SearchContext.test.tsx

```
@@ -0,0 +1,276 @@
+import { SearchProvider, type SearchContextValue } from './SearchContext';
+import { useSearchContext } from './useSearchContext';
+
+// Test component that exposes context values
+function TestConsumer({ onContextValue }: { onContextValue?: (value: SearchContextValue) => void }) {
+  const context = useSearchContext();
+  onContextValue?.(context);
+
+  return (
+    <div>
+      <span data-testid="is-open">{context.isOpen.toString()}</span>
+      <button onClick={context.openSearch} data-testid="open-button">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)